### ISSUE TITLE
--total-num-update unrecognized argument

### ISSUE THREAD

**Issue by wasiahmad** [pytorch/fairseq#2348]
Is `--total-num-update` replaced by `max-update` in the master branch?

Thanks!

**Comment by myleott** [comment]
`--total-num-update` is an option for a specific LR scheduler: https://github.com/pytorch/fairseq/blob/master/fairseq/optim/lr_scheduler/polynomial_decay_schedule.py#L38

`--max-update` is a global option that controls the number of training steps.
